### PR TITLE
Add adamatch HF space and model

### PR DESCRIPTION
**Pull request by johko**
A little import fix and I added Hugging Face model and space links.

### CODE REVIEW

File: examples/vision/adamatch.py [hunk 1]
```diff
@@ -69,7 +69,7 @@
 from tensorflow import keras
 from tensorflow.keras import layers
 from tensorflow.keras import regularizers
-from official.vision.image_classification.augment import RandAugment
+from official.vision.ops.augment import RandAugment
```
Comment (fchollet) [inline examples/vision/adamatch.py:72]: What version of the model garden are you using? Make sure the version is listed in the example.

**Review by fchollet — COMMENTED** [review]
Thanks for the PR

**Comment by fchollet** [comment]
@sayakpaul Hi Sayak, do you have any suggestion on how to reproduce the original results?

Btw it may be beneficial to use the RandAugment version from KerasCV now that it's available.

**Comment by sayakpaul** [comment]
> Btw it may be beneficial to use the RandAugment version from KerasCV now that it's available.

Sure thing. I will try submitting a PR with that. Cc: @LukeWood 

> @sayakpaul Hi Sayak, do you have any suggestion on how to reproduce the original results?

I suspect this to be a version mismatch problem. But @johko mentions that even after 10 runs the performance stays there. Is it the same across different seeds?

**Comment by johko** [comment]
Sorry for the late reply, I was out of office for a bit. I'll run some more experiments across different seeds, as @sayakpaul suggested and will report back on how it worked out.

File: examples/vision/adamatch.py (hunk 1 above)
Comment (LukeWood) [inline examples/vision/adamatch.py:72]: Prefer KerasCV's RandAugment.

```
augmenter = RandAugment(num_layers=2, magnitude=5)
```

can be replaced with:

```
augmenter = keras_cv.layers.RandAugment(layers=3, magnitude=0.5)
```

usage:
```
images = augmenter(images)
```

Comment (johko) [reply examples/vision/adamatch.py]: Thank you Luke, I have adapted the example to KerasCV

**Comment by sayakpaul** [comment]
Awesome. Thanks @johko! Appreciate your willingness.

File: examples/vision/adamatch.py [hunk 2]
```diff
@@ -128,7 +128,7 @@
 
 # Initialize `RandAugment` object with 2 layers of
 # augmentation transforms and strength of 5.
-augmenter = RandAugment(num_layers=2, magnitude=5)
+augmenter = RandAugment(value_range=(0, 255), augmentations_per_image=3, magnitude=0.5)
```
Comment (LukeWood) [inline examples/vision/adamatch.py:131]: augmentations_per_image=2 to reproduce your original, up to you how you want to implement this though.

**Review by LukeWood — APPROVED** [review]
KerasCV usage LGTM

File: examples/vision/adamatch.py (hunk 2 above)
Comment (johko) [reply examples/vision/adamatch.py:131]: Ah, yes I see, the comment above it should be adapted then as well. I give it a try to run it with `augmentations_per_image=2` and see if I am more lucky in getting good results quickly this time, if not I guess I would leave it at `3`

File: examples/vision/md/adamatch.md [hunk 3]
```diff
@@ -26,9 +26,16 @@ be installed using the following command:
...
 
 ```python
-!pip install -q tf-models-official
+!pip install -q tf-models-official==2.9.2
 ```
 
+<div class="k-default-codeblock">
+```
+[33mWARNING: You are using pip version 21.3.1; however, version 22.1.2 is available.
+You should consider upgrading via the '/home/johannes/Projects/keras-io/venv/bin/python -m pip install --upgrade pip' command.[0m
```
Comment (fchollet) [inline examples/vision/md/adamatch.md:35]: Please remove this block

**Review by fchollet — COMMENTED** [review]
Thank you for the update -- looking good!

File: examples/vision/adamatch.py (hunk 2 above)
Comment (LukeWood) [reply examples/vision/adamatch.py]: If you have issues please let me know!

File: examples/vision/md/adamatch.md (hunk 3 above)
Comment (johko) [reply examples/vision/md/adamatch.md]: Sorry for the late reply on this PR as well, I have now updated the code and removed the block

**Comment by fchollet** [comment]
> cla/google Failing after 1s — ❌ Missing CLA from one or more contributors

Please sign the CLA.

**Comment by johko** [comment]
@fchollet this one should be good now as well :)

**Review by fchollet — APPROVED** [review]
LGTM, thank you!

**Pull request merged**